include/wpe/loader.h

```diff
@@ -43,6 +43,12 @@ struct wpe_loader_interface {
     void (*_wpe_reserved3)(void);
 };
 
+void
+wpe_loader_init(const char* library_name);
+
+const char*
+wpe_loader_get_loaded_library_name(void);
```

Comment:
Nit-picking: wpe_loader_get_loaded_implementation_library_name()? Or just wpe_loader_get_implementation_library_name()?

The function name should describe that the library in question is the library that's implementing interfaces, and not some other library.